specification/fabric/Microsoft.Fabric.Management/models.tsp

```diff
@@ -0,0 +1,176 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+using OpenAPI;
+
+namespace Microsoft.Fabric;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@doc("The Microsoft Fabric capacity properties.")
+model FabricCapacityProperties {
+  @doc("The current deployment state of Microsoft Fabric resource. The provisioningState is to indicate states for resource provisioning.")
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+
+  @doc("The current state of Microsoft Fabric resource. The state is to indicate more states outside of resource provisioning.")
+  @visibility("read")
+  state: State;
+
+  @doc("The capacity administration")
+  administration: CapacityAdministration;
+}
+
+@doc("Represents the SKU name and Azure pricing tier for Microsoft Fabric capacity resource.")
+model RpSku {
+  @doc("The name of the SKU level.")
+  name: string;
+
+  @doc("The name of the Azure pricing tier to which the SKU applies.")
+  tier: RpSkuTier;
+}
+
+@doc("The name of the Azure pricing tier to which the SKU applies.")
+union RpSkuTier {
+  string,
+
+  @doc("Fabric tier")
+  Fabric: "Fabric",
+}
+
+@doc("The administration properties of the Fabric capacity resource")
+model CapacityAdministration {
+  @doc("An array of administrator user identities.")
+  members: Array<string>;
+}
+
+@doc("The resource provisioning state.")
+union ProvisioningState {
+  ResourceProvisioningState,
+
+  /** Resource is deleting */
+  Deleting: "Deleting",
+
+  /** Resource is provisioning */
+  Provisioning: "Provisioning",
+
+  /** Resource is updating */
+  Updating: "Updating",
+
+  string,
+}
+
+/** The resource provisioning state. */
+union State {
+  ResourceProvisioningState,
+
+  /** Resource is active */
+  Active: "Active",
+
+  /** Resource is provisioning */
+  Provisioning: "Provisioning",
+
+  /** Resource is updating */
+  Updating: "Updating",
+
+  /** Resource is deleting */
+  Deleting: "Deleting",
+
+  /** Resource is suspending */
+  Suspending: "Suspending",
+
+  /** Resource is suspended */
+  Suspended: "Suspended",
+
+  /** Resource is pausing */
+  Pausing: "Pausing",
+
+  /** Resource is paused */
+  Paused: "Paused",
+
+  /** Resource is resuming */
+  Resuming: "Resuming",
+
+  /** Resource is scaling */
+  Scaling: "Scaling",
+
+  /** Resource is preparing */
+  Preparing: "Preparing",
+
+  string,
+}
+
+@doc("Move resources request payload.")
+model MoveResourcesRequest {
+  @doc("The target resource group where the resources will be moved.")
+  targetResourceGroup: string;
+
+  @doc("The list of resources to move.")
+  resources: Array<string>;
```

Comment:
Can you make this array of armResourceIdentifiers?

Reply:
Don't need to model at all. Removed.